### PR TITLE
Add ftp server template.

### PR DESCRIPTION
**Pull request by baichenxu3**
Add ftp server template.

### CODE REVIEW

**Comment by xiao201208** [comment]
add template to readme and readme.cn

File: documents/ecs/ftp-centos7.yml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+Parameters:
+  FTPUser:
+    Label:
+      zh-cn: FTP服务访问用户名
+      en: FTP server access username
+    Type: String
+    Default: ftptest
+  FTPUserPassword:
+    Label:
+      zh-cn: FTP服务访问用户密码
```
Comment (Prodesire) [inline documents/ecs/ftp-centos7.yml:14]: Please add `。`/`.` to end.

File: documents/ecs/ftp-centos7.yml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            echo "pasv_min_port=50000" >>/etc/vsftpd/vsftpd.conf
+            echo "pasv_max_port=50010" >>/etc/vsftpd/vsftpd.conf
+            echo "pasv_address=127.0.0.1" >>/etc/vsftpd/vsftpd.conf
+            sed -i 's/\#chroot_local_user=YES/chroot_local_user=YES/' /etc/vsftpd/vsftpd.conf
+            sed -i 's/\#chroot_list_enable=YES/chroot_list_enable=YES/' /etc/vsftpd/vsftpd.conf
+            sed -i 's%#chroot_list_file=/etc/vsftpd/chroot_list%chroot_list_file=/etc/vsftpd/chroot_list%' /etc/vsftpd/vsftpd.conf
+            ${CurlCli} -d "{\"status\" : \"SUCCESS\"}"
+          - CurlCli:
+              Fn::GetAtt:
+                - WaitConditionHandle
```
Comment (Prodesire) [inline documents/ecs/ftp-centos7.yml:135]: You can use `${WaitConditionHandle.CurlCil}` directly in the content.

File: documents/ecs/ftp-centos7.yml [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+            echo "pasv_enable=YES" >>/etc/vsftpd/vsftpd.conf
+            echo "allow_writeable_chroot=YES" >>/etc/vsftpd/vsftpd.conf
+            echo "pasv_min_port=50000" >>/etc/vsftpd/vsftpd.conf
+            echo "pasv_max_port=50010" >>/etc/vsftpd/vsftpd.conf
+            echo "pasv_address=127.0.0.1" >>/etc/vsftpd/vsftpd.conf
+            sed -i 's/\#chroot_local_user=YES/chroot_local_user=YES/' /etc/vsftpd/vsftpd.conf
+            sed -i 's/\#chroot_list_enable=YES/chroot_list_enable=YES/' /etc/vsftpd/vsftpd.conf
+            sed -i 's%#chroot_list_file=/etc/vsftpd/chroot_list%chroot_list_file=/etc/vsftpd/chroot_list%' /etc/vsftpd/vsftpd.conf
+            ${WaitConditionHandle.CurlCli} -d "{\"status\" : \"SUCCESS\"}"
+          - FTPUser:
```
Comment (Prodesire) [inline documents/ecs/ftp-centos7.yml:133]: FTPUser and FTPUserPassword can also be used in the content. Drop this dict.

**Pull request merged**